### PR TITLE
Fix popout support for jitsi widgets

### PR DESCRIPTION
**Pull request by dbkr**
* Move URL templating into AppTile so we can use different URLs for iframed widgets and popped-out widgets
 * Remove now-unused get-config call from the widget API

### CODE REVIEW

File: src/utils/AutoDiscoveryUtils.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ import {makeType} from "./TypeUtils";
 import SdkConfig from '../SdkConfig';
 
 const LIVELINESS_DISCOVERY_ERRORS = [
+    AutoDiscovery.ERROR_INVALID,
```
Comment (jryans) [inline src/utils/AutoDiscoveryUtils.js:25]: Hmm, wait why are there changes in the file...? These seem unrelated to Jitsi / widgets.

File: src/utils/WidgetUtils.js [hunk 2]
```diff
@@ -423,32 +391,6 @@ export default class WidgetUtils {
         app.eventId = eventId;
         app.name = app.name || app.type;
 
-        if (app.type === 'jitsi') {
-            console.log("Replacing Jitsi widget URL with local wrapper");
-            if (!app.data || !app.data.conferenceId) {
-                // Assumed to be a v1 widget: add a data object for visibility on the wrapper
-                // TODO: Remove this once mobile supports v2 widgets
-                console.log("Replacing v1 Jitsi widget with v2 equivalent");
```
Comment (jryans) [inline src/utils/WidgetUtils.js:431]: What happened with this v1 -> v2 transformation path? Is it no longer needed?

File: src/utils/AutoDiscoveryUtils.js (hunk 1 above)
Comment (dbkr) [reply src/utils/AutoDiscoveryUtils.js]: oops :(

File: src/utils/WidgetUtils.js (hunk 2 above)
Comment (dbkr) [reply src/utils/WidgetUtils.js:431]: It moved to https://github.com/matrix-org/matrix-react-sdk/pull/4319/files#diff-0e2cb0e79e0d376b964d6d11e52c4c45R563
